src/tasks/AndroidAppBuilder/Templates/CMakeLists-android.txt

```diff
@@ -9,12 +9,19 @@ if(NOT USE_LLVM)
     add_compile_options(-no-integrated-as)
 endif()
 
+# Prevent the warning: shared library text segment is not shareable which is treated as an error
+if (NOT ANDROID_ABI STREQUAL "arm64-v8a")
+    add_link_options(-Wl,--no-warn-shared-textrel)
```

Comment:
do we still need this or is it solved by the `-relocation-model=pic` ?

Reply:
I read that while this fixed a link warning it would actually fail at runtime.

Reply:
@vargaz ^

Reply:
I don't think this is fixed by that. We should keep it for now.